tests/parsing/mod.rs

```diff
@@ -244,3 +245,16 @@ fn parse_long_duration() {
 
     assert_eq!(actual.out, "1min 18sec 797ms");
 }
+
+
+#[rstest]
+#[case("def test [ --a: any = 32 ] {}")]
```

Comment:
ooh ok, good :+1: 

i see this first case is the only one to work now, all the other ones appear to fail :ok_hand: 
let's see if the CI goes green :relieved: